### PR TITLE
Task 1.2

### PR DESCRIPTION
**Pull request by fujoshik**

### CODE REVIEW

File: Task1.2/Program.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    public static string CalculateCheckDigit(string input)
+    {
+        int length = input.Length;
+        int inputNum = int.Parse(input);
+        int sum = 0;
+
+        for (int i = 0; i < length; i++)
+        {
```
Comment (IgorKheidorov) [inline Task1.2/Program.cs:18]: Too complicated and contains unnecessary calculations and methods calls. 
Try
var isbn = Console.ReadLine();
var number = int.Parse(isbn);
for (var i = 2; i <= 10; i++)
{
    var digit = number % 10;
    number /= 10;
    sum += i * digit;
}

File: Task1.2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            sum += (inputNum / (int)Math.Pow(10, length - 1 - i)) * (length + 1 - i);
+            inputNum = inputNum % (int)Math.Pow(10, length - 1 - i);
+        }
+
```
Comment (IgorKheidorov) [inline Task1.2/Program.cs:23]: checkDigit = (11 - (sum % 11))%11;-> to process situation when (sum % 11) = 0;

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]
Please fix according to comments

File: Task1.2/Program.cs (hunk 1 above)
Comment (fujoshik) [reply Task1.2/Program.cs:18]: fixed

File: Task1.2/Program.cs (hunk 2 above)
Comment (fujoshik) [reply Task1.2/Program.cs:23]: fixed

**Review by IgorKheidorov — APPROVED** [review]

**Pull request merged**